docs/guidance/example.md

```diff
@@ -0,0 +1,145 @@
+
+* one or more `locations`, which may be expressed in a variety of ways as [outlined in the specification reference](../../../../reference/#locations). In this example, a motorway junction location is given, using a point location, a gazetteer entry and an address.
+
+```eval_rst
+
+.. jsoninclude:: ../examples/example.json
+   :jsonpointer: /locations
+   :expand:  geometry, gazetteer, address
+   :title: location 1
+
+```
+
+### Budget and budget breakdown
+
+The `budgetBreakdown` array in the `budget` section can be used to provide a breakdown of the budget by period and/or funding source.
+
+In this example, the overall budget for the project covers 3 years and is broken down into amounts per year, with £10,000,000 allocated in 2016, as shown in the first `budgetBreakdown` component.
+
+  ```eval_rst
+
+  .. jsoninclude:: ../examples/example.json
+     :jsonpointer: /budget
+     :expand: amount, budgetBreakdown/0
+     :title: budget
+
+  ```
+
+  ```eval_rst
+
+  .. jsoninclude:: ../examples/example.json
+     :jsonpointer: /budget/budgetBreakdown/0
+     :expand: amount, period, sourceParty
+     :title: budgetBreakdown
+
+  ```
+### Parties (organizations)
+
+The `parties` array is used to provide details about organizations involved in the project and their roles. Organization references elsewhere in the data refer back to entries in this section.
+
+In this example, details are given about the fictional Motorways UK entity, which is referred to from the `sourceParty` section in the `budget breakdown` example above, using the `name` and `id` from the entry in the `parties` section.
+
+```eval_rst
+
+.. jsoninclude:: ../examples/example.json
+   :jsonpointer: /parties/0
+   :expand: address, contactPoint, roles
+   :title: party
+
+```
+
+
+### Documents
+
+The `documents` array is used to provide information and links to documents and documentation relating to the project. During different phases of the project, different document types are expected.
+
+In this example, an environmental impact assessment is provided. The `documentType` field is used to categorise the document against the [document type codelist](../../../../reference/codelists/#documenttype).
+
+```eval_rst
+
+.. jsoninclude:: ../examples/example.json
+   :jsonpointer: /documents/1
+   :expand: address, contactPoint, roles
+   :title: document
+
+```
+### Contracting processes
+
+The `contractingProcesses` array is used to provide information about each contracting process associated with the project, including summary information, a list of modifications and a list of OCDS releases.
+
+This example provides information about a contracting process for the design of the motorway upgrade, with one related document (a tender notice) and 2 related OCDS releases, one for the tender and one for the contract award.
```

Comment:
Use words ("two") for numbers less than 10.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/72/commits/ba8be8d828d38799bb52f3b0cbef655b8f53be6a